tests/scripts/expect/cli-misc.exp

```diff
@@ -164,4 +164,7 @@ expect "0:0:0:0::/64 s low"
 send "route remove ::/64\n"
 expect "Done"
 
+send "diag start\n"
+expect ": InvalidState"
```

Comment:
```suggestion
expect "Error 13: InvalidState"
```
nit

Reply:
I intended to omit the error number in case it changed.